recipes/barril/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "barril" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8b89228988a2bcebf377c56211e96b11a3020184c0a588129fed00fd61766ca4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools_scm
+  run:
+    - attrs >=18.1.0
+    - numpy >=1.11.0
+    - python
+    - ruamel.ordereddict >=0.4.6  # [py27]
+
+test:
+  imports:
+    - barril
+    - barril.basic
+    - barril.basic.format_float
+    - barril.basic.fraction
+    - barril.units
+    - barril.units.scalar_validation
+  requires:
+    - pytest
+    - pytest-mock
+  commands:
+    - pytest --pyargs barril
+
+about:
+  home: https://github.com/ESSS/barril
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Python package to manage units for physical quantities
+
+extra:
+  recipe-maintainers:
+    - igortg
+    - kfasolin
+    - lincrosenbach
+    - nicoddemus
+    - tadeu
```

Comment:
Does everyone agree to be a maintainer?

@tadeu , @lincrosenbach , @kfasolin , @igortg